### PR TITLE
Customer story link is not available anymore [GE Aviation's Digital Group]

### PR DESCRIPTION
**Pull request by v-fearam**
fix: https://github.com/MicrosoftDocs/cloud-adoption-framework/issues/1153

### CODE REVIEW

**Comment by v-fearam** [comment]
#assign-reviewer:hallihan

**Comment by Jak-MS** [comment]
@MicrosoftDocs/cloud-architecture-content-team-pr-reviewers - Please review the proposed changes. If you want to move forward with the content, add a sign-off.

#label:"aq-pr-triaged"
#label:"pending-content-team/business-approval"
Cc: @MicrosoftDocs/public-repo-pr-review-team

File: docs/strategy/motivations.md [hunk 1]
```diff
@@ -68,7 +68,6 @@ The motivations below help IT organizations to focus more on innovation than a m
 - Move towards their [Sustainability goals](./business-outcomes/sustainability.md)
 - Increase business agility.
 - Prepare for new technical capabilities.
-- Build new technical capabilities. Read the [customer story](https://customers.microsoft.com/story/846315-ge-aviation-manufacturing-azure).
```
Comment (BryanLa) [inline docs/strategy/motivations.md:71]: Change to:
- Build new technical capabilities.

**Review by BryanLa — CHANGES_REQUESTED** [review]
@v-fearam - thanks for this update. Just one minor suggestion.

**Comment by BryanLa** [comment]
#label:"cact-review-in-progress"

**Comment by BryanLa** [comment]
CACT #sign-off

**Comment by BryanLa** [comment]
CACT #sign-off
#remove-label:"do-not-merge"
#label:"ready-to-merge"

**Pull request merged**